docs/test-fairy/acct-mgmt/acct-settings.md

```diff
@@ -8,12 +8,132 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
+
+* Slack - See [Slack](https://docs.testfairy.com/Integrations/Slack.html) for information about integrating your Slack account with TestFairy.
+
+* Zendesk - See [ZenDesk](https://docs.testfairy.com/Integrations/Zendesk.html) for information about integrating your Zendesk account with TestFairy.
+
+* Webhooks - You can use TeftFairy webhooks to connect to services. See [Microsoft Teams](https://docs.testfairy.com/Integrations/Microsoft_Teams.html) for more information.
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-5.png')} alt="Account Settings" width="600"/>
+
+### Bug Systems
+You can connect your bug system to TestFairy to report bugs directly to your [Jira Cloud](https://docs.testfairy.com/Bug_Tracking/JIRA_Cloud.html), [GitHub](https://docs.testfairy.com/Bug_Tracking/Github.html), or [Trello](https://docs.testfairy.com/Bug_Tracking/Trello.html) account.
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-6.png')} alt="Account Settings" width="600"/>
+
+### Email Templates
+TestFairy lets you customize the invitation emails it sends. This feature is available only if you use a custom email server (see [SMTP and Gmail](https://docs.testfairy.com/Integrations/SMTP_and_Gmail.html) for more information).
+
+The email is HTML based and can use custom tags.
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-7.png')} alt="Account Settings" width="600"/>
+
+### Security
+* **Require user login before app download** - If you want testers to first login to their tester account prior to downloading your app.
+* **Require [Google] Sign on for all users** - Users must sign in using a Google email address.
+* ``[after adding SAML/Single Sign-on]`` (optional) - Grants access to all apps to testers who sign in with SSO.
+
+#### SAML/Single Sign-On
+Here you can add the [SSO](https://docs.testfairy.com/Single_Sign-On/SSO.html) metadata definitions file. When you add SAML/Single Sign-On, the file will contain your ID, URL and x509 certificate.
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-8.png')} alt="Account Settings" width="600"/>
+
+### <a name="account"></a>Account
+
+The **Account** menu provides access to the following settings:
+
+* First and last name
+* Company name
+* Subdomain
+* User name and password
+* Timezone
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-9.png')} alt="Account Settings" width="600"/>
+
+### Billing
+
+Here you will see your account invoices.
+
+<br/><img src={useBaseUrl('img/test-fairy/acc-settings/acc-set-10.png')} alt="Account Settings" width="600"/>
+
+### Users Whitelist
+
+Allows recording of specific users, instead of recording all user sessions. This is useful in cases in which customer support teams want to record users who are having issues.
+
```

Comment:
```suggestion
Allows recording of specific users instead of recording all user sessions. It helps in cases in which customer support teams want to record users who are having issues.

```